pkg/storkctl/clusterdomainupdate.go

```diff
@@ -0,0 +1,195 @@
+var clusterDomainAliases = []string{"cd"}
+
+func newActivateClusterDomainCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	var allClusterDomains bool
+	var nameClusterDomainUpdate string
+	activateClusterDomainCommand := &cobra.Command{
+		Use:     clusterDomainSubcommand,
+		Aliases: clusterDomainAliases,
+		Short:   "Activate a cluster domain",
+		Run: func(c *cobra.Command, args []string) {
+			activationList := []string{}
+			if allClusterDomains {
+				cdsList, err := k8s.Instance().ListClusterDomainStatuses("kube-system")
+				if err != nil {
+					util.CheckErr(err)
+					return
+				}
+
+				for _, cds := range cdsList.Items {
+					activationList = append(activationList, cds.Status.Inactive...)
+				}
+			} else if len(args) == 1 {
+				activationList = []string{args[0]}
+			} else {
+				util.CheckErr(fmt.Errorf("Exactly one cluster domain name needs to be provided to the activate command"))
+				return
+			}
+			updateUUID := uuid.New()
+			for i, clusterDomainName := range activationList {
+				var name string
+				if len(nameClusterDomainUpdate) > 0 {
+					if len(activationList) > 1 {
+						name = nameClusterDomainUpdate + "-" + strconv.FormatInt(int64(i), 10)
+					} else {
+						name = nameClusterDomainUpdate
+					}
+				} else {
+					if len(activationList) > 1 {
+						name = updateUUID + "-" + strconv.FormatInt(int64(i), 10)
+					} else {
+						name = updateUUID
+					}
+				}
+				clusterDomainUpdate := &storkv1.ClusterDomainUpdate{
+					ObjectMeta: meta.ObjectMeta{
+						Name:      name,
+						Namespace: "kube-system",
+					},
+					Spec: storkv1.ClusterDomainUpdateSpec{
+						ClusterDomain: clusterDomainName,
+						Active:        true,
+					},
+				}
+				_, err := k8s.Instance().CreateClusterDomainUpdate(clusterDomainUpdate)
+				if err != nil {
+					util.CheckErr(fmt.Errorf("Failed to activate cluster domain %v: %v", clusterDomainName, err))
+					return
+				}
+			}
+		},
+	}
+	activateClusterDomainCommand.Flags().BoolVarP(&allClusterDomains, "all", "a", false, "Activate all inactive cluster domains")
+	activateClusterDomainCommand.Flags().StringVar(&nameClusterDomainUpdate, "name", "", "Name for the activate cluster domain action")
+
+	return activateClusterDomainCommand
+}
+
+func newDeactivateClusterDomainCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	var nameClusterDomainUpdate string
+	deactivateClusterDomainCommand := &cobra.Command{
+		Use:     clusterDomainSubcommand,
+		Aliases: clusterDomainAliases,
+		Short:   "Deactivate a cluster domain",
+		Run: func(c *cobra.Command, args []string) {
+			if len(args) == 1 {
+				name := uuid.New()
+				if len(nameClusterDomainUpdate) > 0 {
+					name = nameClusterDomainUpdate
+				}
+				clusterDomainName := args[0]
+				clusterDomainUpdate := &storkv1.ClusterDomainUpdate{
+					ObjectMeta: meta.ObjectMeta{
+						Name:      name,
+						Namespace: "kube-system",
```

Comment:
I think going forward, we shouldn't assume the `kube-system` namespace will be available to use. We should make this configurable instead of hardcording everywhere.